homeassistant/components/assist_pipeline/pipeline.py

```diff
@@ -1352,6 +1361,25 @@ async def execute(self) -> None:
                 assert self.stt_metadata is not None
                 assert stt_processed_stream is not None
 
+                if self.stt_wake_word is not None:
+                    # Avoid duplicate detections by checking cooldown
+                    wake_up_key = f"{self.stt_wake_word.id}"
```

Comment:
Is the ID the right thing to check here, or should we introduce a new property that is "wake word phrase" ?

Reply:
Also, no need to convert to string if it's just the variable

```suggestion
                    wake_up_key = self.stt_wake_word.id
```

Reply:
The `name` property is intended to be used for that. Maybe I should just make this a string rather than trying to reuse the `WakeWord` class?

Reply:
Made this a string instead called `stt_wake_up_key`